packages/ui/src/components/va-button/VaButton.demo.vue

```diff
@@ -1,295 +0,0 @@
-              <va-button color="danger">
-                <va-icon name="gavel" color="textInverted" />
-                Colored icons
-                <va-icon name="shield" color="textInverted" />
-              </va-button>
-              <va-button preset="plain" color="warning">
-                <va-icon name="gavel" color="warning" />
-                Colored icons
-                <va-icon name="shield" color="warning" />
-              </va-button>
-            </va-config>
-          </td>
-        </tr>
-
-        <tr>
-          <td>Outline Buttons</td>
-          <td>
-            <va-button border-color="primary" preset="primary" icon="create">Secondary</va-button>
-            <va-button border-color="primary" preset="secondary" iconRight="clear">Secondary</va-button>
-            <va-button border-color="primary" preset="plain" icon="create" iconRight="clear">
-              Secondary
-            </va-button>
-          </td>
-        </tr>
-
-        <tr>
-          <td>Icons-only Buttons</td>
-          <td>
-            <va-button icon="block" />
-            <va-button preset="primary" icon="block" />
-            <va-button preset="secondary" icon="block" />
-            <va-button preset="plain" icon="block" />
-            <va-button preset="plainOpacity" icon="block" />
-          </td>
-        </tr>
-
-        <tr>
-          <td>
-            Buttons with loading status
-          </td>
-          <td>
-            <va-button :loading="buttonLoading" @click="buttonLoading = true">Start loading</va-button>
-            <va-button preset="primary" :loading="buttonLoading" @click="buttonLoading = true">Start
-              loading
-            </va-button>
-            <va-button @click="buttonLoading = false">Stop loading</va-button>
-          </td>
-        </tr>
-
-        <tr>
-          <td>
-            Round button
-          </td>
-          <td>
-            <va-button round>Label</va-button>
-            <va-button round icon="block" size="small" />
-            <va-button round icon="block" />
-            <va-button round icon="block" size="large" />
-            <va-button preset="primary" round icon="block" />
-            <va-button preset="secondary" round icon="block" />
-            <va-button preset="primary" border-color="warning" round>Label</va-button>
-            <va-button preset="primary" border-color="warning" round icon="block">Label</va-button>
-            <va-button round icon="thumb_up">Label</va-button>
-          </td>
-        </tr>
-
-        <tr>
-          <td>
-            Public methods (focus, blur)
-          </td>
-          <td>
-            <va-button @click="$refs.testButton.focus()">Focus Main Button</va-button>
-            <va-button ref="testButton">Main Button</va-button>
-            <va-button @click="$refs.testButton.blur()">Blur Main Button</va-button>
-          </td>
-        </tr>
-
-        <tr>
-          <td>
-            Multiline text
```

Comment:
<img width="565" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/de3db8f5-4fc1-49bb-875f-2c4c5802a7f6">

I added slightly longer text and it makes story broken.